solutions/instances/outputs.tf

```diff
@@ -2,6 +2,22 @@
 # Outputs
 ##############################################################################
 
+## Cloud logs
+output "cloud_logs_crn" {
+  value       = var.cloud_logs_provision ? module.observability_instance.cloud_logs_crn : null
+  description = "The id of the provisioned Cloud Logs instance."
+}
+
+output "cloud_logs_guid" {
+  value       = var.cloud_logs_provision ? module.observability_instance.cloud_logs_guid : null
+  description = "The guid of the provisioned Cloud Logs instance."
+}
+
+output "cloud_logs_name" {
+  value       = var.cloud_logs_provision ? module.observability_instance.cloud_logs_name : null
+  description = "The name of the provisioned Cloud Logs instance."
+}
+
```

Comment:
What about the outputs for the COS bucket that gets created?

Reply:
Added the cos bucket outputs.